src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+};
+
+export class App extends React.Component<{}, State> {
+  state = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleClick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleClick);
+  }
+
+  handleClick = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
```

Comment:
let's write `[space]` if the `space` is pressed
<img width="201" alt="image" src="https://user-images.githubusercontent.com/55802630/183036389-73e4fa92-898f-42a7-972f-acabeb42fd16.png">

Reply:
also, I have noticed that **fn** on mac os doesn't displayed

Reply:
it's true 😢